client/src/redux/reducers/projects.reducers.js

```diff
@@ -31,6 +31,7 @@ const removeProjectMember = (members, removedMember) => {
 };
 
 const transformMetadata = (project) => {
+  return project;
```

Comment:
Nah it's really annoying that we ever ended up with this weird double-serializer approach instead of just.. refactoring the backend to return a response that's easy to render in React.

Reply:
Yeah agree. Can we kill this function?

Reply:
hes dead jim